go.mod

```diff
@@ -17,8 +17,8 @@ go 1.18
 // gopkg.in/launchdarkly/go-server-sdk.v5 back to "latest" and commit the change.
 
 require (
-	github.com/launchdarkly/go-sdk-common/v3 latest
-	github.com/launchdarkly/go-server-sdk/v6 latest
+	github.com/launchdarkly/go-sdk-common/v3 v3.1.0
```

Comment:
This was on "latest" (not a valid thing to specify) so that the version would float without needing to update it when people run `go build`. At least, I think that's why it was written this way. 

But we can just go ahead and update it going forwards.

Reply:
I went ahead and reverted it to keep it consistent.